app/assets/src/components/ui/controls/GeoSearchInputBox.jsx

```diff
@@ -17,10 +17,15 @@ class GeoSearchInputBox extends React.Component {
       const locationsCategory = "Location Results";
       categories[locationsCategory] = {
         name: locationsCategory,
-        // LiveSearchBox/Search tries to use 'title' as 'key'. Use title + i instead.
-        results: serverSideSuggestions.map((r, i) =>
-          Object.assign({}, r, { key: `${r.title}-${i}` })
-        )
+        // Format title/description for the text box.
+        results: serverSideSuggestions.map((r, i) => {
+          const nameParts = r.name.split(", ");
+          r.title = nameParts[0];
+          // LiveSearchBox/Search tries to use 'title' as 'key'. Use title + i instead.
```

Comment:
What problem are you trying to solve? 
Whats wrong with title in this case? Is it duplicate?

Reply:
Yeah, in the "short name" case there were a lot of duplicates (so changing the list would only remove one) but I'll change this to `loc-{locationiq_id}` instead